src/binary-reader-objdump.cc

```diff
@@ -296,6 +297,16 @@ Result BinaryReaderObjdumpDisassemble::OnOpcodeBare() {
   return Result::Ok;
 }
 
+Result BinaryReaderObjdumpDisassemble::OnOpcodeIndex(Index value) {
```

Comment:
So the idea is that this disassembler can be a little dumb and doesn't need to implment one callback for each opcode, it can just implement OnOpcode and then OnOpcode<ArgType>, so it seems string to have specific  OnOpcode<ArgType>, since we already have OnCallExpr which basically does that for the call opcode specifically.   But I agree that this change is moving towards caring about the specfic opcode...which is counter to that goal.